src/main.cpp.rej

```diff
@@ -0,0 +1,11 @@
+diff a/src/main.cpp b/src/main.cpp	(rejected hunks)
```

Comment:
Looks like a file added by accident. Should add *.rej to `.gitignore` I guess

Reply:
@tklengyel, done!